packages/@lwc/engine-core/src/framework/base-lightning-element.ts

```diff
@@ -70,20 +68,14 @@ function createBridgeToElementDescriptor(
 ): PropertyDescriptor {
     const { get, set, enumerable, configurable } = descriptor;
     if (!isFunction(get)) {
-        if (process.env.NODE_ENV !== 'production') {
-            assert.fail(
-                `Detected invalid public property descriptor for HTMLElement.prototype.${propName} definition. Missing the standard getter.`
-            );
-        }
-        throw new TypeError();
+        throw new TypeError(
+            `Detected invalid public property descriptor for HTMLElement.prototype.${propName} definition. Missing the standard getter.`
+        );
```

Comment:
Why was this code doing this before? I dunno, maybe it was to avoid having a long string in the production bundle? It doesn't seem worth it to me, so I just consolidated it.